hnn_core/tests/test_drives.py

```diff
@@ -314,3 +359,35 @@ def test_add_drives():
                               rate_constant=10.,
                               weights_ampa={'L2_pyramidal': 1.},
                               synaptic_delays={'L5_pyramidal': 1.})
+    with pytest.raises(ValueError, match='probability must be'):
```

Comment:
I realize I have myself done this in the past. But I guess it might be useful to spell out the full message here because I can't distinguish between this test and the "probability is either" one without searching in the codebase for that phrase